media/js/wiki.js

```diff
@@ -518,10 +518,11 @@
         $contributors.on('click', 'a', function(e) {
             var newTab = (e.metaKey || e.ctrlKey);
             var href = this.href;
+            var $li = $(this).parent();
```

Comment:
I think we should either not use a var, and just make it `label: $(this).parent().index()` or make the var `index` and set it to that value, using that var in the data object.  In any case, `$li` isn't really used anywhere and it seems "half-way" to what we actually need, which is weird.